api/types/registry/registry.go

```diff
@@ -102,3 +105,14 @@ type SearchResults struct {
 	// Results is a slice containing the actual results for the search
 	Results []SearchResult `json:"results"`
 }
+
+// ManifestInspect describes the result obtained from contacting the
+// registry to retrieve image metadata
+// TODO(nishanttotla): Decide which other fields to add here
+type ManifestInspect struct {
+	// Digest is the content addressable digest for the image on the registry
+	Digest digest.Digest
```

Comment:
Do you mean marshaling the manifest into `[]byte` before sending? I have a couple of questions
- Why does the digest need to be verified at the client?
- Why does sending `[]bytes` help with it?

> It's ugly to put JSON inside JSON though

This  applies to `ConfigJSON` as well, right?